src/test/osd/TestRados.cc

```diff
@@ -56,10 +56,7 @@ class WeightedTestGenerator : public TestOpGenerator
       oid << m_op;
       if (m_op % 2) {
 	// make it a long name
-	oid << " ";
-	for (unsigned i = 0; i < 300; ++i) {
-	  oid << i;
-	}
+	oid << " " << string(300, 'o');
```

Comment:
@liewegas i don't get it, we append 301 spaces to oid. and we append a string of 301 spaces. so nothing changed in this patch. it's but a refactor unlike claimed by the commit message?